app/controllers/users_controller.rb

```diff
@@ -1,3 +1,26 @@
 class UsersController < ApplicationController
-  def new; end
+  def show
+    @user = User.find(params[:id])
```

Comment:
k tìm thấy user thì sao e.

Reply:
k tìm thấy user thì sẽ báo lỗi record not found ạ

Reply:
thế nó có chết trang k e? e k xử lí nó ah?? 
mà chỗ nào bỏ dc ( thì bỏ đi e nhé

Reply:
Ý chị Ngọc là viết trường hợp check ko tìm thấy đi. VD như không tìm thấy thì chuyền về trang root chẳng hạn. Trong ứng dụng cần phải tránh mọi trường hợp gây nên chết trang em nhé.

Reply:
- Phân biệt find_by vs find
 - Dùng find_by thay cho find